Core/include/Acts/Propagator/detail/PointwiseMaterialInteraction.hpp

```diff
@@ -139,6 +140,10 @@ struct PointwiseMaterialInteraction {
             static_cast<std::underlying_type_t<NavigationDirection>>(nav));
     // put particle at rest if energy loss is too large
     nextP = (mass < nextE) ? std::sqrt(nextE * nextE - mass * mass) : 0;
+    // minimum momentum below which we will not push particles via material
+    // update
+    static constexpr double minP = 50 * Acts::UnitConstants::MeV;
```

Comment:
should we make this configurable? not necessarily in this pr but we can also follow up with a ticket

Reply:
In the end it's an "epsilon". We can also just make it smaller.

Reply:
Decreased to 10 MeV.

Reply:
but it relates to if we want to stop particles or not right? so to me it looks like we are slightly changing the physics here. also the `ParticlesStopped` aborter will not fire anymore I guess? https://github.com/acts-project/acts/blob/main/Core/include/Acts/Propagator/StandardAborters.hpp#L206

Reply:
@paulgessinger one followup question

Reply:
The thing is, if this if not >0, the Stepper update has an floating-point exception and we can't really proceed. So Yeah, this is not ideal, but I don't see another way, other than hard terminating the propagation right here.